cmake/compiler_settings.cmake

```diff
@@ -46,7 +46,7 @@ if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
   add_compile_options(/wd4068) # Silence warning C4068: unknown pragma
   add_compile_options(/wd5030) # Silence warning C5030: unknown gnu/clang attribute
   add_compile_options(/W4)     # Display all other un-silenced warnings
-  add_compile_options(/bigobj) # Increase .obj sections, needed for hard coded pre-verified hashes
+  # add_compile_options(/bigobj) # Increase .obj sections, needed for hard coded pre-verified hashes
```

Comment:
Why remove everything from `preverified_hashes_mainnet.cpp`? Doesn't seem necessary for this PR.

Reply:
This is still a draft and the commit will be reverted when ready for merge.
I had to do this cause with all preverified hashes it takes ages on my machine to build both the windows binary and the linux one.